src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs

```diff
@@ -45,10 +55,15 @@ public override int GetHashCode()
         {
             if (!_hashComputed)
             {
-                _hashCode = (AssemblyInfo ?? string.Empty + Target?.FullName ?? string.Empty).GetHashCode();
+                _hashCode = (AssemblyInfo ?? string.Empty + PackageId ?? string.Empty).GetHashCode();
```

Comment:
Can you replace this with a better hash builder? See https://stackoverflow.com/questions/263400/what-is-the-best-algorithm-for-an-overridden-system-object-gethashcode for example

Reply:
Replaced the hash builder.